lib/hmmlearn/hmm.py

```diff
@@ -906,34 +906,16 @@ def _initialize_sufficient_statistics(self):
         stats['post_mix_sum'] = np.zeros((self.n_components, self.n_mix))
         stats['post_sum'] = np.zeros(self.n_components)
 
-        # The following statistics are stored in lists so we can
-        # accumulate chunks of data for multiple sequences (aka
+        lambdas, mus = self.means_weight, self.means_prior
+        stats['m_n'] = lambdas[:, :, None] * mus
```

Comment:
I think this, as well as the update to m_n in accumulate_sufficient_statistics, only needs to be done `if 'm' in self.params`?

Reply:
you're right. done